recipes/graphviz/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "2.38.0" %}
+
+package:
+  name: graphviz
+  version: {{ version }}
+
+source:
+  fn:  graphviz-{{ version }}.tar.gz
+  url: http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-{{ version }}.tar.gz
+  md5: 5b6a829b2ac94efcd5fa3c223ed6d3ae
+  patches:
+    # This patch was submitted upstream:
+    # http://www.graphviz.org/mantisbt/view.php?id=2448
+    - gvconfig_libdir.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  detect_binary_files_with_prefix: True
+
+requirements:
+  build:
+    - cairo  # [linux]
+    - expat  # [linux]
+    - libpng  1.6*  # [unix]
+    - libtiff  4.0.*  # [unix]
+    - libtool  # [linux]
+    - freetype  2.5.*  # [unix]
+    - jpeg  9*  # [unix]
+    - pango  >=1.39.0  # [linux]
+    - pkg-config  # [osx]
```

Comment:
Because we need the pkg-config from the system instead of the conda one to find the `gtk2-devel` files.